tdrs-backend/tdpservice/users/models.py

```diff
@@ -175,6 +175,11 @@ def is_deactivated(self):
         """Check if the user's account status has been set to 'Deactivated'."""
         return self.account_approval_status == AccountApprovalStatusChoices.DEACTIVATED
 
+    @property
+    def region_or_stt(self) -> models.Model:
```

Comment:
minor nitpick, but could this property be named `location` instead? also we don't typically declare types in our codebase - not saying i'm against it.

Reply:
Thats a better name! And I wasnt sure if we wanted the declaration or not. Just added in case. Removed since it doesnt happen elsewhere.